libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -259,6 +259,40 @@ bool AP_RPM::arming_checks(size_t buflen, char *buffer) const
     return true;
 }
 
+float AP_RPM::get_average_rpm() const
+{
+    float rpm_avg = 0.0f;
+    uint8_t valid_escs = 0;
```

Comment:
```suggestion
    uint8_t valid_sensors = 0;
```

Reply:
Committed a fix for this and the other spot where I used valid_rpms instead of valid_sensors. Now it's consistent!